go.mod

```diff
@@ -1,6 +1,11 @@
 module github.com/kubev2v/migration-planner
 
-go 1.22.10
+go 1.23.6
+
+// toolchain is automatically added after upgrate go version to 1.23
+// This was done as result of upgrfading github.com/openshift/client-go
+// This is used to enforce a specific version of the Go toolchain
```

Comment:
That's strange that client-go from 2023 needs go version 1.24 which was relesed this year. Can't the reason be different?

Reply:
Ok, this is true
This was added because of my local go version
I removed it totally for now